{"uuid": "4128c5a4-1dfb-4fb1-bd1a-cc7c604aaa05", "children": ["448376e8-c3cf-4cd3-bf78-447e556cb87a", "0322bd9b-3221-4437-bf9b-7e96a7df8384", "b68aad64-d28e-4f2d-a92c-abb6f9ef581d", "57f3450d-dfa2-4523-9376-69e483f6ccba", "93d5142d-727b-4528-836a-2b9b455d03df", "d144c7dc-2d82-415e-ae40-c890190140b7"], "befores": [{"name": "setup_profiles", "status": "failed", "statusDetails": {"message": "Failed: AP latest and active are different\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/interOp/conftest.py\", line 228, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_ref,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 571, in setup_profiles\n    pytest.fail(\"AP latest and active are different\")\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/outcomes.py\", line 153, in fail\n    raise Failed(msg=msg, pytrace=pytrace)\n"}, "attachments": [{"name": "Active Config: ", "source": "63bf8fd7-4e06-47db-9347-5d8191e66c9b-attachment.attach"}, {"name": "Latest Config: ", "source": "27df0adb-ae0b-43d9-bd6b-6a2c3d087b24-attachment.attach"}, {"name": "FAILURE: AP Logs: ", "source": "bc24a240-a49c-4fb7-8ee5-a348b9bb4ae7-attachment.attach"}], "start": 1639084439077, "stop": 1639084761303}], "start": 1639084439077, "stop": 1639084763164}